dockerfiles/Dockerfile.browser

```diff
@@ -16,6 +16,7 @@ COPY packages ./packages
 RUN npm ci
 RUN npm run build
 
+RUN npm pack --verbose packages/core/
```

Comment:
core packages is now needed as generated types refer back to `@bugsnag/core`. Without this the latest publish version of `@bugsnag/core` will be fetched from npm